modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/background/task/BackgroundTaskExecutorConfigurator.java

```diff
@@ -46,9 +46,6 @@ protected void activate(BundleContext bundleContext) {
 
 		_registerBackgroundTaskExecutor(
 			bundleContext, reindexPortalBackgroundTaskExecutor);
-
```

Comment:
Please put this removal change in another commit and explain the reasons for that